e2e/test/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -3,33 +3,47 @@ import {
   popover,
   appBar,
   restore,
+  setTokenFeatures,
+  describeEE,
 } from "e2e/support/helpers";
 import { ORDERS_QUESTION_ID } from "e2e/support/cypress_sample_instance_data";
 
-describe("scenarios > embedding > full app", () => {
+describeEE("scenarios > embedding > full app", () => {
   beforeEach(() => {
     restore();
     cy.signInAsAdmin();
+    setTokenFeatures("all");
```

Comment:
Even though these tests don't really check for the token (they merely rely on the existence of an iframe which simulates full-app embedding), I wanted to hint that this can only run on enterprise instance with a premium token.